solarforecastarbiter/reports/main.py

```diff
@@ -207,6 +207,10 @@ def create_raw_report_from_data(report, data):
     Returns
     -------
     raw_report : datamodel.RawReport
+
+    Todo
```

Comment:
does this section render ok? I'm not aware of support for a todo section in numpydoc

Reply:
It doesn't seem that reports haven't been added to the api.rst to create the toctree. I'll have to add that first before I check but. 

I've seen todo sections in some google style but perhaps it isn't in numpydocs style.  I'll  check.